### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dahifi**
@timkpaine I'm not going to finalize this merge; my hope was that we would hold off on changes to master until we're ready for a version bump. It looks like you've been busy lately, and I wanted to ask that we hold off with changes to master. 

One of the changes that you made, about three weeks ago, to the place order function, broke my code, and I was hoping we could wait until keep track of any code-breaking changes to to the method signatures until they could be properly communicated. 

Just my $0.02.

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# TDA Config vars
```
Comment (timkpaine) [inline .env.default:1]: remove file

File: docs/authentication.md [hunk 2]
```diff
@@ -26,12 +26,21 @@ Set the name and id to something reasonably unique, and the redirect to `http://
...
-run `tdameritrade.auth.authentication` from a python prompt, with `client_id=consumer_key` and `redirect_uri` from step 4.
+Run `tdameritrade.auth.authentication` from a python prompt, with `client_id=consumer_key` and `redirect_uri` from step 4.
 
 ### Step 6
 Sign in and authorize your app.
 
-You can enter credentials by hand or store them in environment variables `TDAUSER` and `TDAPASS`. When stored, the first page will be filled in and advanced automaticly. 
+You can enter credentials by hand. 
+
+Or store them in environment variables in `.env`:
```
Comment (timkpaine) [inline docs/authentication.md:36]: we can enumerate the env vars but don't include the file

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+setuptools~=46.1.3
```
Comment (timkpaine) [inline requirements.txt:1]: no requirements file, remove, don't pin versions

File: tdameritrade/auth/auth_form_executioner.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
+import time
+
+import requests_oauthlib
+
+
+class FormExecutioner:
+    tda_auth_url = 'https://auth.tdameritrade.com/auth'
+    tda_refresh_token_url = 'https://api.tdameritrade.com/v1/oauth2/token'
+
+    def __init__(
```
Comment (timkpaine) [inline tdameritrade/auth/auth_form_executioner.py:10]: does this work when no security questions are prompted?

File: tdameritrade/auth/tda_config_provider.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (timkpaine) [inline tdameritrade/auth/tda_config_provider.py:4]: no import side effects

**Comment by dahifi** [comment]
@danielgrzenda I'm gonna need you on this one to help resolve.

**Comment by dahifi** [comment]
I tagged Daniel, but it looks like it's actually the commits from Clay Herendeen that are failing. The chromedriver fails on my end; can't find the username CSS. 

```
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[id="username"]"}
  (Session info: headless chrome=81.0.4044.113)
```

His user profile isn't gone. I tried reaching out to him via LinkedIn. 

Has anyone else been able to get his auth function to work? I may have to branch off of my last known good branch and cherrypick the other commits to get us caught back up. 

Time to brush up on my git rebasing.

File: tdameritrade/auth/auth_form_executioner.py (hunk 4 above)
Comment (dahifi) [reply tdameritrade/auth/auth_form_executioner.py]: No and entire branch was reverted.

**Comment by dahifi** [comment]
@timkpaine I reverted @ka05's branch. I couldn't get it to work and struck the whole branch out. I see you were having issues with it as well on e903bb. 

I'll make a new branch for this and start and issue to track completion on it. 

Will leave the close to you until we can give it a version bump.

**Pull request merged**